### DISCUSSION TITLE
Encoder Interface issue

### DISCUSSION THREAD

**Discussion by PE-07** [shanteacontrols/OpenDeck#53] [Q&A]
I'm trying to interface a rotary optical encoder with open deck .I couldn't find any optical encoder that works on voltage supply les than 5V .But the open deck  works on 3.3V logic. Do I need to provide external 5v supply to encoders and then translate the signals to 3.3V or is there any other method for direct interfacing?

**Comment by PE-07** [comment]
I'm using open deck V 2.0.1

**Reply by paradajz** [reply]
You will need external power supply, yes, since there isn't any +5V pin on the board. However, all OpenDeck inputs are 5V tolerant so you won't have any issue with that at least.

**Reply by PE-07** [reply]
Thanks!!!
